release/release_tests.yaml

```diff
@@ -3673,3 +3673,51 @@
 
     type: sdk_command
     file_manager: sdk
+
+- name: chaos_dataset_shuffle_push_based_sort_1tb
+  group: core-dataset-tests
+  working_dir: nightly_tests
+  legacy:
+    test_name: chaos_dataset_shuffle_push_based_sort_1tb
+    test_suite: chaos_test
+
+  stable: false
+
+  frequency: nightly
+  team: core
+  cluster:
+    cluster_env: shuffle/shuffle_app_config.yaml
+    cluster_compute: shuffle/datasets_large_scale_compute_small_instances.yaml
+
+  run:
+    timeout: 7200
+    prepare: ' python setup_chaos.py --node-kill-interval 1200 --max-nodes-to-kill 3'
+    script: RAY_DATASET_PUSH_BASED_SHUFFLE=1 python dataset/sort.py --num-partitions=1000 --partition-size=1e9
+    wait_for_nodes:
+      num_nodes: 20
+    type: sdk_command
+    file_manager: sdk
+
+- name: chaos_dataset_shuffle_sort_1tb
+  group: core-dataset-tests
+  working_dir: nightly_tests
+  legacy:
+    test_name: chaos_dataset_shuffle_sort_1tb
+    test_suite: chaos_test
+
+  stable: false
+
+  frequency: nightly
+  team: core
+  cluster:
+    cluster_env: shuffle/shuffle_app_config.yaml
+    cluster_compute: shuffle/datasets_large_scale_compute_small_instances.yaml
+
+  run:
+    timeout: 7200
+    prepare: ' python setup_chaos.py --node-kill-interval 1200 --max-nodes-to-kill 3'
```

Comment:
`dataset_shuffle_sort_1tb_small_instances` currently takes ~3000s: https://buildkite.com/ray-project/release-tests-branch/builds/568#9e27bb9e-3d08-4e87-af3e-db1f70c4d644/783-813
Should we use a smaller kill interval?